src/docs/WSL_BUILD.md

```diff
@@ -0,0 +1,25 @@
+# Local development build [Ubuntu Linux / WSL2]
+
+Make sure you have the dependencies.
+- Install Python 
+```
+sudo apt update
+sudo apt install python3 python3-pip
+```
+- Add the following lines to your ~/.bashrc file: 
+```
+alias python=python
+alias pip=pip3
+```
+- Checkout Marlin
+```
+git clone git@github.com:MarlinFirmware/Marlin.git Marlin
```

Comment:
```suggestion
git clone git@github.com:MarlinFirmware/Marlin.git Marlin -b bugfix-2.0.x --depth 1
```
This will just check out the most recent single commit, which is much faster. Then you won't need these next three lines.

Reply:
Good catch! I'll make the change right away.

Reply:
When I work on this, I constantly rm -rf that directory and clone it again. It's nice to not have to download the whole history.

Reply:
My approach is to do
```
git checkout .
git reset --hard
git clean -f
```